Tools/setup/arch.sh

```diff
@@ -0,0 +1,127 @@
+#! /usr/bin/env bash
+
+## Bash script to setup PX4 development environment on Arch Linux.
+## Tested on Manjaro 18.0.1.
+##
+## Installs:
+## - Common dependencies and tools for nuttx, jMAVSim
+## - NuttX toolchain (omit with arg: --no-nuttx)
+## - jMAVSim simulator (omit with arg: --no-sim-tools)
+##
+## Not Installs:
+## - Gazebo simulation
```

Comment:
Yer, that one is a pain.

Reply:
Is there easier than build from source? Do you have steps even if it's only a draft?

Reply:
That'll probably be solved in the next version of this script.

Reply:
It's doable using `yay`, for now, let's leave it out.